app/models/favorite.rb

```diff
@@ -17,12 +17,14 @@ def favorite_count
     @content.values.sum
   end
 
-  def favorite_delete(id)
-    @content.delete(id.to_s)
+  def favorite_delete(pets)
```

Comment:
Refactors delete method so it can iterate through an array of ids (like from an application)